web/ui/react-app/src/pages/agent/Agent.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { FC } from 'react';
+
+export const AgentContent: FC = () => {
```

Comment:
I don't think there's a good reason to put this into a separate component from `Agent`, since this component doesn't load anything. I'd just return this directly from `Agent`.